runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -1001,6 +1001,26 @@ TR_SharedCacheRelocationRuntime::getCurrentLockwordOptionHashValue(J9JavaVM *vm)
    return currentLockwordOptionHashValue;
    }
 
+OMRProcessorDesc
+TR_SharedCacheRelocationRuntime::getProcessorDescription(TR_FrontEnd *fe, J9VMThread *curThread)
```

Comment:
Should this be named something like `getProcessorDescriptionFromSCC` to be clear what it does? We could get the processor description of current machine. Also, I don't see where this is used.

Reply:
It's used over here:
https://github.com/eclipse/openj9/blob/43a903dcc369d4c40661626b7868d61d20a1d5ee/runtime/compiler/control/rossa.cpp#L1917

When we have an existing SCC and passes the AOTHeader validation, then we will copy processor features from SCC to relocatableTarget.

I will change it to `getProcessorDescriptionFromSCC` to avoid confusion with the other `getProcessorDescription()` we have for TR::CPU.